infrastructure/variables.tf

```diff
@@ -15,3 +16,117 @@ variable "stage" {
 variable "region" {
   default = "us-east-1"
 }
+
+variable "host_ip" {
+  # This will be overwritten.
+  default = "127.0.0.1"
+}
+
+variable "database_user" {
+  default = "drpostgresuser"
+}
+
+variable "database_password" {
+  # This will be overwritten by the password in terraform.tfvars.
+  # It's kept there so it's secret.
+  default = "drpostgrespassword"
+}
+
+variable "django_secret_key" {
+  # TODO: generate a new one of these and store it in terraform.tfvars as well.
+  default = "NtG1bxZU115GThwrLuAJe0PhTVN9hJ4P"
+}
+variable "django_debug" {
+  default = "False"
+}
+variable "database_port" {
+  default = "5432"
+}
+
+variable "database_timeout" {
+  default = "30"
+}
+
+variable "running_in_cloud" {
+  default = "True"
+}
+
+# This is a placeholder until there is a production image ready.
+variable "workers_docker_image" {
+  default = "wkurt/dr_worker:3"
+}
+variable "foreman_docker_image" {
+  default = "wkurt/dr_foreman:4"
+}
+variable "use_s3" {
+  default = "True"
+}
+variable "local_root_dir" {
+  default = "/home/user/data_store"
+}
+variable "raw_prefix" {
+  default = "raw"
+}
+variable "temp_prefix" {
+  default = "temp"
+}
+variable "processed_prefix" {
+  default = "processed"
+}
+
+# Output our production environment variables. These should be in
+# parity with the env files such as workers/environments/dev.
+output "environment_variables" {
+  value = [
+    {name = "REGION"
+      value = "${var.region}"},
+    {name = "USER"
+      value = "${var.user}"},
+    {name = "STAGE"
+      value = "${var.stage}"},
+    {name = "AWS_ACCESS_KEY_ID_WORKER"
+      value = "${aws_iam_access_key.data_refinery_user_worker_key.id}"},
+    {name = "AWS_SECRET_ACCESS_KEY_WORKER"
```

Comment:
Why was this changed? Is there also a secret_access_key_not_worker?

Reply:
Yes, the box's key. When we run `export AWS_SECRET_ACCESS_KEY=XXX` so it can be formatted into the job specs, the rest of the script then has that env var set so the next time we run `terraform apply` (at the end) we get errors about how the data refinery worker user doesn't have enough permissions to do all things terraform wants to do (including just refreshing some resources).